rancher2/structure_cluster_v2_rke_config_certificate_rotation.go

```diff
@@ -0,0 +1,41 @@
+package rancher2
+
+import rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
+
+func flattenClusterV2RKEConfigRotateCertificates(in *rkev1.RotateCertificates) []interface{} {
+	if in == nil {
+		return nil
+	}
+
+	obj := make(map[string]interface{})
+
+	if in.Generation > 0 {
```

Comment:
Seems like, from `lines 12` and `16` generation and services don't have to be set for a cert rotation to be successful. Is that true?
Sorry for all the questions :)

Reply:
This function is for parsing the response from the rancher api and transforming it to terraform.  Given the certificate rotation object is nilable, we may not have a value to transform to tf.